00_hello_world/main_test.go

```diff
@@ -18,7 +18,7 @@ func TestMainOutput(t *testing.T) {
 	main()
 
 	// Then
-	expected := strconv.Quote("Hallo du schöne Welt!\n")
+	expected := strconv.Quote("Hallo du schöne Welt!\n1\n1\n2\n3\n5\n8\n13\n")
```

Comment:
This could be:
```go
	expected := strconv.Quote("Hallo du schöne Welt!
1
1
2
3
5
8
13
`)
```
It's not unequivocally the right thing to do, however. It improves readability in one dimension, but hampers in another by breaking the indentation structure. It's a judgment call, really.

Reply:
Done. Thanks.